src/main/scala/nl/knaw/dans/easy/stage/EasyStageDataset.scala

```diff
@@ -108,7 +110,7 @@ object EasyStageDataset {
     val relativePath= getDatasetRelativePath(folder).toString
     for {
       sdoDir <- mkdirSafe(getSDODir(folder))
-      fis     = FileItemSettings(s.sdoSetDir, s.ownerId, getDatasetRelativePath(folder).toFile, None, None, None)
+      fis     = FileItemSettings(s.sdoSetDir, null, s.ownerId, getDatasetRelativePath(folder).toFile, None, None, None, None)
```

Comment:
null -> None, perhaps rather implement another apply method that fills in these None's in FileItemSettings now that you are submitting mutual dependent pull requests anyway.